File: CursoPOOUber/Csharp/Account.cs
using System;

namespace AccountNS
{

    //definition of the "Account" class.
    public class Account
    {
        //Attributes
        public int Id { get; set; }
        public string Name { get; set; } = "";
        public int Document { get; set; }
        public string Email { get; set; } = "";

        public string Password { get; set; } = "";

        public Account(string name, int document) 
        {
            this.Name = name;
            this.Document = document;

        }
        
        public virtual void PrintDataDriver()
        {
            Console.WriteLine("These are the deatils of this account: " + "Name: " + Name + " Document: " + Document + " E-mail: " + Email + " Password: " + Password);
        }

        public static implicit operator Account(string v)
        {
            throw new NotImplementedException();
        }
    }

}

File: CursoPOOUber/PHP/Account.php
<?php
//Definition of the "Account" class.
class Account {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;

    public function __construct($name, $document) {
        $this->name = $name;
        $this->document = $document;
    }

    public function printDataDriver() {
        echo "Name: " + $this->name + " Document: " + $this->document + " E-mail: " + $this->email + " Password: " + $this->password;
    }
}
?>